types/src/on_chain_config/consensus_config.rs

```diff
@@ -8,6 +8,9 @@ use move_core_types::account_address::AccountAddress;
 use serde::{Deserialize, Serialize};
 use std::collections::HashMap;
 
+/// Default Window Size for Execution Pool
+pub const DEFAULT_WINDOW_SIZE: Option<u64> = Some(1u64);
```

Comment:
Hmm.. we need to decide if it's `Some(1)` or `None` (to disable by default). If we use `Some(1)`, I'll need to update this PR: https://github.com/aptos-labs/aptos-core/pull/15869